IFC4x3/Properties/a/AssetCriticality_3gS4Hpk8L3vgoq4ExCulT5/Documentation.md

```diff
@@ -1 +1 @@
-Rating of the assets critically to the operation of the facility
+Rating of the asset's critically to the operation of the facility
```

Comment:
Something is of here. Can 'critically' be a noun?

Reply:
should be criticality then it is a noun!